cmd/yourapp/internal/templates/base.templ

```diff
@@ -1,24 +1,30 @@
 package templates
 
+import "fmt"
+
 templ HeadSection(title string) {
 	<head>
 	if len(title) == 0 {
 		<title>Your App!</title>
 	} else {
 		<title>Your App! - {title}</title>
 	}
-	<link rel="stylesheet" href="/static/main.css" />
-	<link rel="icon" href="/static/icon.svg" />
+	<link rel="stylesheet" href={prefixString("/static/main.css")} />
+	<link rel="icon" href={prefixString("/static/icon.svg")} />
```

Comment:
I would have expected link href to need a `templ.SafeURL` rather than a string. This might be a bug.

Reply:
Posed the question of whether this is intended [here](https://github.com/a-h/templ/issues/1091)

Reply:
Clarified the templ issue a bit, but I don't want to hold up this PR for that.